clarifai/cli/model.py

```diff
@@ -78,21 +140,47 @@ def init(model_path, model_type_id, github_pat, github_repo, branch, local_ollam
 
     MODEL_PATH: Path where to create the model directory structure. If not specified, the current directory is used by default.
     """
-    # Handle the --local-ollama-model flag
-    if local_ollama_model:
-        if github_repo or branch:
-            raise click.ClickException(
-                "Cannot specify both --local-ollama-model and --github-repo/--branch"
-            )
-        github_repo = "https://github.com/Clarifai/runners-examples"
-        branch = "ollama"
-
     # Resolve the absolute path
     model_path = os.path.abspath(model_path)
 
     # Create the model directory if it doesn't exist
     os.makedirs(model_path, exist_ok=True)
 
+    # Validate parameters
+    if port and not port.isdigit():
+        raise click.BadParameter("Port must be a number")
```

Comment:
```suggestion
        logger.error("Invalid value: --port must be a number")
        raise click.Abort()
```
Use logger to keep the logs clean and consistent and use `click.Abort()` to exit CLI